docker/pixl-python/Dockerfile

```diff
@@ -16,7 +16,8 @@ SHELL ["/bin/bash", "-o", "pipefail", "-e", "-u", "-x", "-c"]
 
 ARG TEST="false"
 
-RUN export DEBIAN_FRONTEND=noninteractive && \
+RUN <<EOF
+    export DEBIAN_FRONTEND=noninteractive && \
```

Comment:
if we're doing EOF, do we need the trailing `\`?

Reply:
Ah good point; my IDE adds them back in quite aggresively.